ui/appui-react/src/appui-react/layout/footer/message-center/MessageCenterField.tsx

```diff
@@ -0,0 +1,155 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Notification
+ */
+import * as React from "react";
+import {
+  Button,
+  NotificationMarker,
+  Popover,
+  Tabs,
+} from "@itwin/itwinui-react";
+
+import { SvgChat } from "@itwin/itwinui-icons-react";
+import type { MessageType } from "@itwin/core-react";
+import { Icon } from "@itwin/core-react";
+import { UiFramework } from "../../../UiFramework";
+import { OutputMessagePriority } from "@itwin/core-frontend";
+import { MessageCenterMessage } from "./MessageCenterMessage";
+import { MessageManager } from "../../../messages/MessageManager";
+import { TitleBar } from "../dialog/TitleBar";
+import type { NotifyMessageDetailsType } from "../../../messages/ReactNotifyMessageDetails";
+import "./MessageCenterField.scss";
+
+/** Message Center Field React component.
+ * @public
+ */
+export const MessageCenterField: React.FC = () => {
+  const [messages, setMessages] = React.useState(MessageManager.messages);
+  const [notify, setNotify] = React.useState("");
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const indicator = React.createRef<HTMLDivElement>();
+  const title = UiFramework.translate("messageCenter.messages");
+
+  const handleOpenChange = (isOpenState: boolean) => {
+    setNotify("");
+    setIsOpen(isOpenState);
+  };
+
+  const isProblemStatus = (message: NotifyMessageDetailsType): boolean => {
+    // See priority values in DgnPlatform defined in NotificationManager
+    return (
+      message.priority === OutputMessagePriority.Error ||
+      message.priority === OutputMessagePriority.Fatal
+    );
+  };
+
+  const notifyStatus = () =>
+    messages.some((msg) => isProblemStatus(msg)) ? "negative" : "primary";
+
+  const handleMessagesUpdatedEvent = () => {
+    setNotify(notifyStatus());
+    setMessages(MessageManager.messages);
+  };
+
+  React.useEffect(() => {
+    MessageManager.onMessagesUpdatedEvent.addListener(
+      handleMessagesUpdatedEvent
+    );
+    MessageManager.registerAnimateOutToElement(indicator.current);
+
+    return () => {
+      MessageManager.onMessagesUpdatedEvent.removeListener(
+        handleMessagesUpdatedEvent
+      );
+    };
+  });
+
+  const getMessages = (tab: string): React.JSX.Element[] => {
+    return [...messages]
+      .reverse()
+      .map((details: NotifyMessageDetailsType, index: number) => {
```

Comment:
(FYI: I have resolved previous comments that were not addressed, I assume it's because GH sometimes decides to collapse unresolved comments). Use implicit types.
```suggestion
      .map((details, index) => {
```